src/main/java/com/neuvector/DockerRunCommandBuilder.java

```diff
@@ -23,17 +23,9 @@ public DockerRunCommandBuilder(String runtime) {
         cmdList.add("--rm");
     }
 
-    DockerRunCommandBuilder withUserAndGroup(String userIdGroupId) {
```

Comment:
We do not need the userAndGroup anymore. We can run the container with the default user and if it is root - it is ok.. This was something we added since the scan_result.json was being owned by the user and the customer did not like it was owned by root. So we were passing in the id of the caller instead of root. Since we do not have scan_result.json anymore stored in the users file system (we read it in the container, before we destroy the container) it can be owned by the root.